admin_guide/managing_networking.adoc

```diff
@@ -155,15 +155,26 @@ the same rules apply for claiming a domain for a namespace based on the creation
 time of the object. For example, the oldest route winning against any other
 claimants wanting to use the same namespace.
 
-For example, ownership of routes is based on the claims made for that host name with the same namespace. The oldest route wins against any other claims. 
+For example, ownership of routes is based on the claims made for that host name
+with the same namespace. The oldest route wins against any other claims.
+
+[NOTE]
+====
+Routes and ingress objects have the same data structure internally on the
+router, with ingress objects having the capability to turn into multiple
+objects. The claim rules above apply for each of these objects, but, because an
+ingress object can have two hostnames, one hostname can be claimed by the
+ingress object and be active, but the other might not be able to claim, and be
+ignored.
```

Comment:
@bfallonf This is not clear to me (but, maybe that's OK). "one hostname can be claimed by the
ingress object and be active, but the other might not be able to claim, and be ignored."
What does "be ignored" mean here? If one host name is ignored, does that mean anything for the other? (Sorry if this is an ignorant question.)

Reply:
s/claim/claimed ??

Reply:
@mburke5678 True.

@knobunc Would it be more accurate to say "The claim rules above apply for each of these objects. However, because an ingress object can have two hostnames, one hostname can be claimed by the ingress object and be active, while the second cannot claim, and remains inactive and ignored."

Reply:
Remains inactive is probably better.